src/main/webapp/event.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <meta charset="UTF-8">
+    <title>CodeU 2019 Starter Project</title>
+    <link rel="stylesheet" href="/css/main.css">
+    <link rel="stylesheet" href="/css/event.css">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Cabin">
+    <script src="/js/navigation-loader.js"></script>
+    <script src="/js/event.js"></script>
+</head>
+
+<body onload="addLoginOrLogoutLinkToNavigation();" onload="buildUI()">
+    <nav id="navigation">
+        <br>
+        <br>
+        <br>
+        <ul>
+            <a href="/ ">
+                <li>HOME</li>
+            </a>
+        </ul>
+    </nav>
+    <div class="content">
+
+        <div class="title">
+            <p id="event-title"> Event Title</p>
+        </div>
+
+        <div class="details">
+            <p id="username" class="username" style="font-family:Nunito">Author:</p>
```

Comment:
Do you need in-line css or can that be put in the css file?

Reply:
Sorry, no!

Reply:
+1, let's put it in the css file.

Reply:
Done!